src/documentation/markdown.rs

```diff
@@ -382,4 +418,107 @@ And a line, or a rule."##;
             });
         assert_eq!(v.first(), Some(&(12..14)));
     }
+
+    #[test]
```

Comment:
might be worth to write a macro to generate the test case bodies, if you need help here, let me know :)

Reply:
yes, I would need help. Do you have any examples already in the code with macro+tests?

Also, do you mean like parametrized tests where I can add many many tests and the expected results and run once? This would be nice! I do not know how to do it in rust but I can try if I have some direction  ^‿^

Reply:
See below, I think macros are not even necessary in this case :)